mingw-w64-rrdtool/PKGBUILD

```
@@ -0,0 +1,48 @@
+_realname=rrdtool
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.7.2
+pkgrel=1
+pkgdesc="Round Robin Database (mingw-w64)"
+arch=('any')
+url='https://oss.oetiker.ch/rrdtool/index.en.html'
+license=('GPL2')
+makedepends=("autoconf" "automake" "libtool")
```

```suggestion
makedepends=("${MINGW_PACKAGE_PREFIX}-autotools"
                          "${MINGW_PACKAGE_PREFIX}-cc")
```

Please try with this.